src/LuigisBoxBundle/Service/SearchRequest.php

```diff
@@ -45,8 +45,11 @@ public function search(SearchUrlBuilder $searchUrlBuilder): SearchResponse
         try {
             $request = $this->searchFactory->prepareRequest($searchUrlBuilder);
 
+            $url = $searchUrlBuilder->toUrlQuery();
+            Assert::notEmpty($url);
+
             return new SearchResponse(
-                $searchUrlBuilder->toUrlQuery(),
+                $url . '&v=' . $this->getUniqueStamp(),
```

Comment:
Wouldn't appropiate request header be better than cache busting link? Or are headers not supported?

Reply:
This is no my idea.
From documentation:
```
 Be aware that there is an HTTP cache in front of the API and you may receive cached responses. If you want to be sure that you are seeing fresh data, append a cache-busting parameter to the API URL, e.g. &v=1. You may need to increment the number for subsequent requests to avoid the cache repeatedly.
```